### DISCUSSION TITLE
How do I load a file saved with -w?

### DISCUSSION THREAD

**Discussion by tddschn** [mitmproxy/mitmproxy#5382] [General]
I ran `mitmproxy -w outfile`, exited, and wanted to load `outfile` later with `mitmproxy`, how do I do that?

I tried the `-r` option, but it didn't work since `outfile` is not a flow file.

**Comment by mhils** [comment]
`-r` is indeed the correct approach, outfile should be a flow file.

**Reply by tddschn** [reply]
I knew that, but I couldn't find an option to save to a flow file except from the mitmweb GUI menu.
